Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1217,6 +1218,9 @@
 /* message status if message delivery to a recipient is skipped. We skip delivering group messages to users who have left the group or unregistered their Signal account. */
 "MESSAGE_STATUS_RECIPIENT_SKIPPED" = "Skipped";
 
+/* Label indicating that a message failed to send. */
+"MESSAGE_STATUS_SEND_FAILED" = "Send Failed";
```

Comment:
This is in the footer, but the translation comments show "MESSAGE_STATUS_FAILED" as a message footer. Are they used in slightly different contexts? Or can one of them be deleted?